src/AuditManager.php

```diff
@@ -52,11 +52,9 @@ public function getConfiguration()
     }
 
     /**
-     * NEXT_MAJOR: Use `\Doctrine\ORM\EntityManagerInterface` for argument 1.
-     *
      * @return AuditReader
      */
-    public function createAuditReader(EntityManager $em)
+    public function createAuditReader(EntityManagerInterface $em)
```

Comment:
This is a BC break however. Classes which are extending `AuditReader` and overwriting `createAuditReader` will break due to this change.

Hence we also planned to do this in the next major release as the comment above suggested.

Reply:
Sure I understand. I'm closing the PR. Thank you !